File: snippet1.js
function calculateAverage(numbers) {
    let sum = 0;
    
    for (let i = 0; i <= numbers.length; i++) { 
        
        
        sum += numbers[i]
    }
    return sum / numbers.length; 
}
debugger
const numbers = [10, 20, 30, 40, 50];
const avg = calculateAverage(numbers);
console.log(`The average is: ${avg}`);

File: snippet1.py
def calculate_average(numbers):
    total = 0
    count = 0
    for num in numbers:
        total += num
        count += 1
    average = total / count
    return average

numbers = [10, 20, 30, 40, 50]
average = calculate_average(numbers)
print("The average is:", average)